src/Core/Collections/CustomDropdown.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from "react";
+
+import "./_custom-dropdown.scss";
+
+export default class CustomDropdown extends Component {
+    onCollectionChange = (event) => {
+        this.props.onChange(event.target.value)
+    }
+    render() {
+        const {optionArray} = this.props;
+        return (
+            <div className={"custom-dropdown-wrapper"}>
+                <div className={"shadow"} />
+                <select className={"select-collections"}
+                        onChange={this.onCollectionChange}>
+                    {optionArray.map(option =>
```

Comment:
It would be better if you make this mapping with a render function. Let's render the mapping on a new renderer function.